### PR TITLE
Temporary fix for #294: only build R and Python RSE volumes automatically.

### PR DESCRIPTION
**Pull request by gvwilson**
1.  Rename the `rse` target in `Makefile` to `r-rse` to accurately reflect what it does (it builds the R RSE volume, not both of the RSE volumes).
2.  Modify `.travis.yml` to only build the R and Python RSE volumes for now while we try to figure out the broken `datetime` manipulation described in #295.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -110,7 +110,7 @@ py : _book/py/index.html _book/py/rse.pdf
 py-rse : _book/py-rse/index.html _book/py-rse/py-rse.pdf
 
 ##   r-rse      : rebuild RSE R HTML and PDF.
-rse : _book/r-rse/index.html _book/r-rse/r-rse.pdf
+r-rse : _book/r-rse/index.html _book/r-rse/r-rse.pdf
```
Comment (lwjohnst86) [inline Makefile]: duh, forgot this. Thanks for the catch.

File: .travis.yml [hunk 2]
```diff
@@ -15,7 +15,7 @@ addons:
 
 script:
   - Rscript -e "tinytex::install_tinytex()"
-  - make everything
+  - make r-rse py-rse
```
Comment (lwjohnst86) [inline .travis.yml]: @gvwilson I have a fix as per my comment in #295, if you can revert this change, we can merge this edit it.

Comment (gvwilson) [reply .travis.yml]: Change reverted. Please try to get a fix for the data file in place before Tuesday's meeting - thanks.
